policy/aggregation_type.go

```diff
@@ -214,6 +265,38 @@ func (aggTypes AggregationTypes) IsValidForTimer() bool {
 	return true
 }
 
+// Quantiles returns all the quantiles found in the list aggregation types.
+func (aggTypes AggregationTypes) Quantiles(p pool.FloatsPool) []float64 {
```

Comment:
`PooledQuantiles` might be a better name? Also how often is this method going to be called?